### PR TITLE
Add N/A option to a life stage

### PR DESCRIPTION
**Pull request by sabrine33**

### CODE REVIEW

File: src/pages/registration/SectionForm.tsx [hunk 1]
```diff
@@ -48,6 +48,16 @@ type SectionFormParams = {
...
 
+export const LifeStageMap: Record<string, string> = {
+  ...Object.entries(LifeStage).reduce(
+    (acc, [key, value]) => {
+      acc[key] = value;
+      return acc;
+    },
+    {} as Record<string, string>
+  ),
+  'N/A': 'n/a'
```
Comment (khelwood) [inline src/pages/registration/SectionForm.tsx:59]: What do the two versions of N/A represent here?

Comment (sabrine33) [reply src/pages/registration/SectionForm.tsx:59]: n/a for not applicable, I suppose, that's the ticket's request

Comment (khelwood) [reply src/pages/registration/SectionForm.tsx:59]: Yes, why are there two versions 'N/A' and 'n/a'? What do the two different versions represent?

Comment (sabrine33) [reply src/pages/registration/SectionForm.tsx:59]: oh, as the uppercase is for label, and the lowercase is for key, just to match how the other life stage values are set.

**Review by khelwood — COMMENTED** [review]
The core expects null rather than `'n/a'` as a life stage value. Is there somewhere in here that translates `'n/a'` to null?

File: src/pages/registration/SectionForm.tsx (hunk 1 above)
Comment (sabrine33) [reply src/pages/registration/SectionForm.tsx:59]: I checked it with the core and it looks like LifeStage is a required field. I am updating the branch, to send null to the core, but probably you need to do some work at your end.

Comment (khelwood) [reply src/pages/registration/SectionForm.tsx:59]: Yep there is a pull request in core to make LifeStage nullable. 👍

**Review by khelwood — APPROVED** [review]

**Comment by sabrine33** [comment]
merged to devel

**Pull request merged**